integrations/google/internal/vars/vars.go

```diff
@@ -0,0 +1,6 @@
+package vars
+
+type Vars struct {
+	OAuthData string `var:"secret"`
+	JSON      string `var:"secret"`
```

Comment:
1. What does `var` mean in the struct hints?
2. Why do both of them have the var name `"secret"`?

Reply:
this is documented in https://github.com/autokitteh/autokitteh/blob/0db4458ddd2dc0546ba746f47cbd9e47f388c156/sdk/sdktypes/vars.go#L50